essay.tex

```diff
@@ -11,35 +11,71 @@
+It will start of by explaining the importance of Sprints in Scrum for games development and then go into why Sprints can be of different lengths to then argue on whether there is an ideal length for sprints to last, it will further go on to arguing that 1 week long sprints could be quite a ideal time lengths for small teams, and it will finally end with a conclusion of the argument and essay.
 
-\section{Your section title here}
 
+\section{So let's talk about sprints}
+\iffalse
 Write the main body of your essay here. Add more sections if appropriate. You may choose to write about each of your three papers in its own section, or you may choose a different structure. Either way, remember that you are being assessed on technical insight and analysis: it is not enough to merely summarise the contents of the three papers. You must demonstrate the ability to make inferences beyond what is written in the papers, and to draw the three papers together into a single coherent narrative.
 
 Your essay must make a clear recommendation, in terms of which of the three techniques you have reviewed is the best according to whichever metric or metrics you feel is most appropriate. You must justify your choice, backing it up with empirical evidence. However remember that an academic essay is not a murder mystery: you should already have briefly discussed your recommendation in the introduction and in other parts of the essay. Do not save it for a grand reveal at the end.
+\fi
+Developing digital games is hard and it gets even harder when you need to develop games with a team, so much more can go wrong when multiple people are working together on the same project, this is why a lot of people within the games industry practice Agile development methods; such as Scrum.
+\\~\\
+Scrum has an incremental development process known as Sprints, Sprints are pre-planned time periods in which the team has decided on what they will work on during the sprint. Sprints are a very important part of the Scrum methodology as they give a specific time limit for the team to work on tasks to implement into the project \cite{five} \cite{eight}.\\
+At the end of a Sprint, a sprint review meeting occurs in which the team gets together to discuss what happened during the sprint and update their project to the latest build of the game with all the new features and fixes that were made during the sprint\cite{eight}.\\
+Some developers like to use sprint reviews as a way of updating their fans on what they have been working on lately for the game, other developer who have early access games might even update their game after each sprint to give the fans regular new features and bug fixes in order to keep the game fresh for the players.
+\\~\\
+This is what the Agile manifesto says about sprints ``Deliver working software frequently, from a couple of weeks to a couple of months, with a preference to the shorter timescale.``\cite{three}.
```

Comment:
For quotations, please provide a page number. For sources that are not paginated, please indicate where the sentence is located (e.g., para. 2, line 3).